ballerina/records.bal

```diff
@@ -78,7 +78,7 @@ public enum Protocol {
 
 # Configurations related to Kafka authentication mechanisms.
 #
-# + mechanism - Type of the authentication mechanism. Currently, SASL_PLAIN and SCRAM are supported
+# + mechanism - Type of the authentication mechanism. Currently only SASL_PLAIN is supported
```

Comment:
```suggestion
# + mechanism - Type of the authentication mechanism. Currently only `SASL_PLAIN` is supported
```

Reply:
Added in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/377/commits/d8e897a4fcf01ef0602e9e87794a77441a554f65